src/Core_Sideloader_Studio/UniversalAutoResolver/Core.UAR.Studio.cs

```diff
@@ -446,6 +461,8 @@ internal static void ResolveStudioBGM(ExtensibleSaveFormat.PluginData extData, R
             if (extData != null && extData.data.ContainsKey("bgmInfoGUID"))
             {
                 string bgmGUID = (string)extData.data["bgmInfoGUID"];
+                string bgmAuthor = (string)extData.data["bgmInfoAuthor"];
+                string bgmWebsite = (string)extData.data["bgmInfoWebsite"];
```

Comment:
Oops, one of these days I'll remember dictionaries store nulls and therefore throw exceptions on not found instead of return null...but not this day...